controllers/comments.controllers.js

```diff
@@ -29,3 +31,15 @@ exports.postCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id: commentId } = req.params;
+
+  Promise.all([deleteComment(commentId), checkCommentIdExists(commentId)])
```

Comment:
don't need to check if it exists!

What does node-postgres tell you if it doesn't delete anything?